platforms/chibios/flash.mk

```diff
@@ -23,6 +23,10 @@ define EXEC_DFU_UTIL
 	$(DFU_UTIL) $(DFU_ARGS) -D $(BUILD_DIR)/$(TARGET).bin
 endef
 
+define EXEC_DFU_FW_UPDATER
+	DfuFwUpdater_cli -D $(BUILD_DIR)/$(TARGET).bin
```

Comment:
does this support dfu-util, rather than a custom/proprietary flasher?

Reply:
We don't support dfu-util and use our own flasher: DfuFwUpdater_cli.exe , Where should this flasher be submitted ?

Reply:
Is this tool open source?

Reply:
We will open source and will upload to github today.

Reply:
That appears to be windows only?

Reply:
@drashna  Yea, currently only windows are supported.

Reply:
So what can people do if they want to flash their board from a Linux or macOS machine?

Reply:
@fauxpark Currently, we hope users to use Windows to flash their board.

Reply:
> @fauxpark Currently, we hope users to use Windows to flash their board.

As somebody that doesn't use Windows for their desktop machine anymore, this leaves a sour taste in my mouth. 

Since the whole of QMK supports Linux, MacOS and Windows, adding a windows only tool doesn't feel right.  And you are dumping the work on the community and QMK to create a linux and macOS utility.